### PR TITLE
ProperMotion API and UI improvements

### PR DESCRIPTION
**Pull request by cquiroz**
This PR uses the updated properMotion field from the API and adds these UI improvements:

* Properly position the error field if search is empty
* While searching disable all proper motion fields

### CODE REVIEW

File: common/src/main/resources/graphql/schemas/ObservationDB.graphql [hunk 1]
```diff
@@ -131,7 +131,7 @@ input CreateSiderealInput {
   ra: RightAscensionInput!
   dec: DeclinationInput!
   epoch: EpochString
-  properVelocity: ProperVelocityInput
+  properMotion: ProperMotionInput
```
Comment (rpiaggio) [inline common/src/main/resources/graphql/schemas/ObservationDB.graphql:134]: I hope you updated this file by using the shell script.

Comment (cquiroz) [reply common/src/main/resources/graphql/schemas/ObservationDB.graphql:134]: Please expand 😄

File: targeteditor/src/main/scala/explore/targeteditor/TargetBody.scala [hunk 2]
```diff
@@ -86,9 +94,6 @@ object TargetBody {
             targetPropsL,
             { case (n, t, _ /*ms*/ ) =>
               EditSiderealInput.name.set(n.value.some) >>> TargetQueries.UpdateSiderealTracking(t)
-            // >>> EditSiderealInput.magnitudes.set( ms.map(m =>
-            //        MagnitudeInput(m.value.toBigDecimal, m.band, none, m.system.some)
-            //      ))
```
Comment (rpiaggio) [inline targeteditor/src/main/scala/explore/targeteditor/TargetBody.scala:91]: This was here waiting for the API to support it.

Comment (cquiroz) [reply targeteditor/src/main/scala/explore/targeteditor/TargetBody.scala:91]: Ah sorry I can restore it

**Review by rpiaggio — APPROVED** [review]
LGTM

File: targeteditor/src/main/scala/explore/targeteditor/TargetBody.scala (hunk 2 above)
Comment (rpiaggio) [reply targeteditor/src/main/scala/explore/targeteditor/TargetBody.scala:91]: I should have put a comment

File: common/src/main/resources/graphql/schemas/ObservationDB.graphql (hunk 1 above)
Comment (rpiaggio) [reply common/src/main/resources/graphql/schemas/ObservationDB.graphql:134]: There's a `./fetchODBSchema.sh` that calls the API via instrospection to get the schema and updates the file in the project.

**Pull request merged**